style.css

```
@@ -0,0 +1,610 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+  font-family: 'Poppins', sans-serif;
+  margin: 0px;
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  width: 1320px;
+  position: relative;
+}
+
+.container_top {
+  position: relative;
+  height: 670px;
+  background-color: #f6f9fb;
+  z-index: 5;
+}
+
+.bubble21 {
+  position: absolute;
+  height: 780px;
+  right: 0px;
+  top: -30px;
+}
+
+.banner {
+  position: absolute;
+  width: 900px;
+  right: -40px;
+  top: 120px;
+}
+
+.baner_text {
+  position: absolute;
+  padding: 20px;
+  width: 490px;
+  font-size: 15px;
+  top: 230px;
+}
+
+.baner_title {
+  font-size: 53px;
+  font-weight: 500;
+  margin-bottom: 40px;
+}
+
+.darkmode {
+  position: fixed;
+  background-color: #5965e9;
+  width: 60px;
+  height: 60px;
+  border-top-right-radius: 50px;
+  border-bottom-right-radius: 50px;
+  top: 200px;
+  display: flex;
+  align-items: center;
+  z-index: 99;
+}
+
+.darkmode_icon {
+  width: 25px;
+  margin-left: 22px;
+}
+
+.header {
+  position: fixed;
+  display: flex;
+  width: 1320px;
+  justify-content: flex-end;
+  z-index: 99;
+}
+
+.header {
+ padding-bottom: 5px;
+}
+
+#header_logo {
+  margin: 10px 130px 0px 10px;
+}
+
+#header_logo_dark {
+  margin: 10px 130px 0px 10px;
+}
+
+.digital {
+  width: 140px;
+}
+
+.header_item {
+  margin: 15px 20px 15px 20px;
```

```suggestion
  margin: 15px 20px;
```
To samo, tylko skrócone.